jpa/org.eclipse.persistence.jpa/src/main/java/org/eclipse/persistence/internal/jpa/StoredProcedureQueryImpl.java

```diff
@@ -557,7 +566,12 @@ public List getResultList() {
                 if (hasMoreResults()) {
                     if (isOutputCursorResultSet) {
                         // Return result set list for the current outputCursorIndex.
-                        List results = (List) getOutputParameterValue(getCall().getOutputCursors().get(outputCursorIndex++).getName());
```

Comment:
Interesting. This appears to have always worked (or it was a bug before my fix) because I didn't make this change.

Reply:
This same fix may be needed in getSingleResult() as well.